cli/console.ts

```diff
@@ -404,7 +404,7 @@ function printExecutedActionErrors(
   executionAction: dataform.IExecutionAction
 ) {
   const failingTasks = executedAction.tasks.filter(
-    task => task.status === dataform.ActionResult.ExecutionStatus.FAILED
+    task => task.status === dataform.TaskResult.ExecutionStatus.FAILED
```

Comment:
This is a bugfix/type fix I assume? In theory it's breaking as the enums have different values.

Reply:
Yeah, I'm surprised this wasn't throwing an error before - the new type is definitely right: https://github.com/dataform-co/dataform/blob/5995f4e169c2972db6207b44e4e45b55d5a20bcd/protos/execution.proto#L98